datumaro/plugins/open_images_format.py

```diff
@@ -0,0 +1,155 @@
+# Copyright (C) 2021 Intel Corporation
+#
+# SPDX-License-Identifier: MIT
+
+import contextlib
+import csv
+import glob
+import json
+import os.path as osp
+import re
+
+from datumaro.components.extractor import (
+    AnnotationType, DatasetItem, Importer, Label, LabelCategories, Extractor,
+)
+
+# image IDs in Open Images are always hexadecimal numbers
+_RE_IMAGE_ID = re.compile(r'[0-9a-fA-F]+')
+
+class _OpenImagesPath:
```

Comment:
These classes are not supposed to be private.

Reply:
Changed to public; mind explaining why, though?

Reply:
"Access modifiers" doesn't make much sense for plugins. We only looks for specific symbols in plugins + _technically_, such bookkeeping dataset information might be useful outside.